packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js

```diff
@@ -143,6 +147,8 @@ export const IAMPolicyDescription = () => {
     const intl = useIntl();
 
     const s3Bucket = formOptions.getState().values.billing_source?.data_source?.bucket;
+    const aliasesEnabled = formOptions.getState().values.aws?.aliases?.enabled;
+    const orgUnitsEnabled = formOptions.getState().values.aws?.org_units?.enabled;
```

Comment:
I think we can call `formOptions.getState()` just once

Reply:
No problem! Make sense! :+1: